src/parser.c

```diff
@@ -559,3 +559,23 @@ size_t pos_to_str_with_limit(struct pos_array *pos, uint8_t *str, size_t limit)
     }
     return len;
 }
+
+int pos_is_zero(struct pos_array *pos)
```

Comment:
Redis uses `strtoll` which is available in C99 for error handling. Both of us just missed it!